_includes/samples/PriceFeeds/PriceConverter.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.7;
 import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
 
 /**
- * Network: Rinkeby
+ * Network: Rinkeby (No Goerli Feed for EUR/USD at this time)
```

Comment:
what impact will this have on our docs?

Reply:
The priceconverter example remains on Rinkeby. Alternatively we can point it to mainnet feeds but we want developers to be able to test it. If we can get some forex feeds on Goerli, that would help.